packages/x-components/src/x-installer/api/base-api.ts

```diff
@@ -118,4 +118,14 @@ export class BaseXAPI implements XAPI {
       console.warn('We know X is awesome, but you only need to initialize it once.');
     }
   }
+
+  /**
+   * Closes the Application.
+   *
+   * @public
+   */
+  close(): void {
+    this.bus?.emit('UserClickedCloseEventsModal');
```

Comment:
You can remove this event, in the case of `oysho` you need to emit ºUserClickedCloseEventsModal` to make it work. It is because we are using a `<BaseEventsModal>`. 

I don't know why we are using that component because it can be replaced by `MainModal`. In any other case,  `<BaseEventsModal>`  component has a prop to listen new events, you we can modify these events to listen to the `UserClickedCloseX`

Reply:
Yes, I think we can remove it too. In Oysho's branch where the `x` packages versions are updated, we are already using `MainModal` so it is fine if we just emit `UserClickedCloseX`.